st2actions/st2actions/notifier.py

```diff
@@ -120,12 +122,17 @@ def _post_generic_trigger(self, liveaction):
                    'status': liveaction.status,
                    'start_timestamp': str(liveaction.start_timestamp),
                    'action_name': liveaction.action,
```

Comment:
Action trigger uses action_name. When I wrote the notifier, I didn't want to break the behavior for action trigger. For notification trigger, I went with action_ref because that's the right thing to do :P. We could have both action_ref and action_name for action trigger and deprecate action_name at some point?